Gems/Atom/RHI/Code/Source/RHI/Factory.cpp

```diff
@@ -78,8 +83,19 @@ namespace AZ
                     }
                 }
             }
-#endif // defined(USE_RENDERDOC)
+#endif
 
+#if defined(USE_PIX)
+            if (AZ_TRAIT_PIX_MODULE && !s_pixModule)
```

Comment:
We probably want a command line flag to prevent loading the DLL if USE_PIX is defined but we don't actually want to enable a GPU capture for this run (e.g. when we want to use Pix for doing an instrumented CPU profile). Otherwise, change looks pretty good!